### ISSUE TITLE
Request for new option to avoid crossing top surface

### ISSUE THREAD

**Issue by tytouf** [alexrj/Slic3r#1634]
Hi, during the print of a large part with an honeycomb infill, the printing of an intermediate "roof" showed some sort of swelling, probably due to the air that gets trapped in the honeycomb cell is getting warmer. This can be seen at the back of the part as a honeycomb pattern:
![_igp1376](https://f.cloud.github.com/assets/1037063/1797581/504eaab8-6af4-11e3-8870-a6c7172a36bc.JPG)
The problem is that due to this swelling, the print head scraped the bumps when passing over this "roof".
I guess a work around is to increase the infill value to prevent such bumps but as I said the part is large and that would extend the printing time quite a lot.
In summary, it would be great if 
an option similar to "Avoid crossing perimeters" could exist for such cases.

**Comment by SparkyCola** [comment]
I've had this problem too and think a little Z-Move could be enough/appropriate.
Plus driving all the way around might take longer in a lot of cases (since you often can't move trough the not-perimeters which makes the way around can be pretty long (depending on size an shape of the part)).

**Comment by lordofhyphens** [comment]
@SparkyCola Z hop is already available through retraction settings, fyi :)
